javatests/arcs/sdk/jvm/DummyEntity.kt

```diff
@@ -11,11 +11,12 @@
 
 package arcs.sdk.jvm
 
+import arcs.core.common.Referencable
 import arcs.sdk.Entity
 import arcs.sdk.EntitySpec
 
 /** Fake [Entity] implementation. */
-data class DummyEntity(val text: String) : Entity {
+data class DummyEntity(override val id: String, val text: String) : Entity, Referencable {
```

Comment:
Should all of the generated Entity classes implement Referencable? If so, I can give you some pointers to the code generator class, it shouldn't be too hard to add.

Reply:
With the current implementation of `CrdtSet`, `CrdtSingleton` and `CrdtEntity`, the answer to this seems to be yes, but I still have gaps in my understanding here.

Should I expect that the `internalId` that's part of an entity on the `arcs.sdk.Entity` side will be the same thing as an `id` that the CRDTs use to uniquely identify an item?

Reply:
I'm not 100% sure what internalId is. I think it's just the unique ID for the entity, which in the case of wasm gets decoded from the wasm bridge. For jvm we'll need to generate it ourselves.

I _think_ this is the same as the CRDT's id.